cmd/snap-confine/udev-support.c

```diff
@@ -306,9 +306,6 @@ void sc_setup_device_cgroup(const char *security_tag)
 		return;
 	}
 
-	/* Note that -1 is the neutral value for a file descriptor.
-	 * The cleanup function associated with this variable closes
-	 * descriptors other than -1. */
```

Comment:
Why is this comment removed?

Reply:
It was misplaced. I should have removed it when I refactored the device cgroup support to a separate file.